lib/praxis/request.rb

```diff
@@ -4,15 +4,25 @@ class Request
     attr_reader :env, :query
     attr_accessor :route_params, :action
 
+    PATH_VERSION_PREFIX = "/v".freeze
+    CONTENT_TYPE_NAME = 'CONTENT_TYPE'.freeze
+    PATH_INFO_NAME = 'PATH_INFO'.freeze
+    REQUEST_METHOD_NAME = 'REQUEST_METHOD'.freeze
+    QUERY_STRING_NAME = 'QUERY_STRING'.freeze
+    API_VERSION_HEADER_NAME = "HTTP_X_API_VERSION".freeze
+    API_VERSION_PARAM_NAME = 'api_version'.freeze
+    API_NO_VERSION_NAME = 'n/a'.freeze
+    VERSION_USING_DEFAULTS = [:header,:params].freeze
+    
```

Comment:
moving them out of the inline reduces the allocations, that's why I did it. There's no need to create these objects every time. (I was doing some memory tests and saw that, so I decided to change it)